tests/testthat/test-util.R

```diff
@@ -6,3 +6,21 @@ test_that("null-or-value works", {
   expect_equal(NULL %||% NULL, NULL)
   expect_equal(NULL %||% 2, 2)
 })
+
+test_that("binrary serialisation is transitive", {
+  f <- function(x, identical = TRUE) {
+    y <- bin_to_object(object_to_bin(x))
+    is_equivalent_to <- if (identical) is_identical_to else equals
```

Comment:
This has clearly come from something old of mine, as this is old testthat now :) - this line can be deleted

Reply:
Yeah, taken this from redux :)